.pre-commit-config.yaml

```diff
@@ -1,14 +1,17 @@
 repos:
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v3.1.0"
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: "v5.0.0"
     hooks:
-      - id: prettier
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+        args: ["--unsafe"]
```

Comment:
fixes `check-yaml` hook for mkdocs `pymdownx` extensions starting with `!!`

https://github.com/pre-commit/pre-commit-hooks/issues/552